system/Router/RouteCollection.php

```diff
@@ -807,7 +807,8 @@ public function resource(string $name, array $options = null): RouteCollectionIn
 		// resources are sent to, we need to have a new name
 		// to store the values in.
 		$new_name = ucfirst($name);
-
+		// replace / as \
+        $new_name = str_replace("/","\\",$new_name);
```

Comment:
Can we change it to one-liner which will `ucfirst` all parts of the namespace?
```php
$new_name = implode("\\", array_map('ucfirst', explode('/', $name)));
```

Reply:
Nice, I'm sorry that due to regional reasons (China), there was a problem with VPN, which was not timely.